domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParser.java

```diff
@@ -7,19 +7,29 @@
 
 @Component
 public class ShowConditionParser {
+    private static final String AND_CONDITION_REGEX = "\\s\\.AND\\.\\s";
+    private static final String AND_OPERATOR = " .AND. ";
```

Comment:
I've asked Steve and there's no particular reason why this has to be .AND. instead of a more conventional AND. Let's change to AND please, will make the conditions more readable

Reply:
replaced .AND. with AND